### DISCUSSION TITLE
I'm unable to enable HTTPS for my domain

### DISCUSSION THREAD

**Discussion by prajwalit** [community/community#119751] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm unable to enable HTTPS for my domain - Prajwalit.com. It says the DNS records are misconfigured. But I'm not able to figure out what's wrong.

![image](https://github.com/community/community/assets/155951/70e74118-a0f5-482f-83fd-60037ac04583)

![image](https://github.com/community/community/assets/155951/b274bbf4-1422-4386-b8c1-d86336e4f36f)

Can someone please help?
Thank you!

**Comment by yoannchaudet** [comment]
You have one extra IPv6 in your list. Remove `2606:50c0:8004::153` ;)

**Comment by prajwalit** [comment]
You're a lifesaver! Thank you so much!
